### ISSUE TITLE
Need help with minification

### ISSUE THREAD

**Issue by TauseefMalik** [prateekkalra/Selection-js#30]
I get token errors on using a minifier. Can anybody please help?

**Comment by tpscrpt** [comment]
@TauseefMalik yes! It's because the selection.js script uses es6, which isn't compatible with a lot of minification tools (which use the standard es2015, or es5). You should be able to find it soon with
```html
<script src="https://unpkg.com/selection-js"></script>
```
To automatically get a minified version in your projects.
